install/templates/pass_auth.sh.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+sed -i s/^PasswordAuthentication.*/PasswordAuthentication\ yes/ /etc/ssh/sshd_config
+systemctl restart sshd
+cat /etc/ssh/sshd_config
+echo "Enabled PasswordAuthentication"
+{% if ansible_distribution_major_version == '8' %}
+sudo dnf clean all
+sudo dnf install {{ epel_repo.el8 }}  -y
+sudo dnf module install -y python36
+sudo alternatives --set python /usr/bin/python3
+sudo dnf --enablerepo=PowerTools install perl-Data-UUID -y
```

Comment:
Why do you need to explicitly install `perl-Data-UUID` here and not for CentOS 7?  Don't you have this requirement elsewhere?

Reply:
I had seen this requirement in the Pbench installation documentation. But I just checked the doc and can't find it now

Reply:
The pbench-agent RPM should have its own dependencies.  I'd suggest removing this to confirm.